Documentation/project-docs/developer-guide.md

```diff
@@ -226,7 +226,11 @@ msbuild System.Runtime.Tests.csproj /p:TargetGroup=netfx
 
 The tests can also be filtered based on xunit trait attributes defined in [`xunit.netcore.extensions`](https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions). These attributes are specified above the test method's definition. The available attributes are:
 
-_**`OuterLoop`:**_
+#### OuterloopAttribute
+
+```cs
+OuterLoop()
```

Comment:
why not 
```c#
[Outerloop]
```
which is how everyone will use it. Simmilarly `[` `]` around other attributes below.